source/declarations.tex

```diff
@@ -2562,37 +2551,33 @@
 \end{example}
 
 \pnum
-In a declaration
-\opt{\grammarterm{attribute-specifier-seq}}
-\tcode{T}
-\tcode{D}
-where
-\tcode{D}
-is an unadorned identifier the type of this identifier is
-``\tcode{T}''.
+In a declaration \tcode{T} \tcode{D}
+where \tcode{D} has the form
+\begin{ncsimplebnf}
+declarator-id \opt{attribute-specifier-seq}
+\end{ncsimplebnf}
+the type of the \grammarterm{declarator-id}
+in \tcode{D} is ``\tcode{T}''.
+The optional \grammarterm{attribute-specifier-seq}
+appertains to the entity declared by \tcode{D}.
```

Comment:
We don't need the "In a declaration" part here. How about reducing this to:

> A *declarator* of the form *declarator-id* *attribute-specifier-seq*[opt] determines the type of its declared entity to be `T`. The optional *attribute-specifier-seq* appertains to the declared entity.

Reply:
Its true that its not quite needed, however it makes it consistent with the rest of [dcl.meaning]

Reply:
See my comments on maintaining `T` through subclauses